### PR TITLE
Added java code for generating fibonacci number

### PR DESCRIPTION
**Pull request by sagarduwal**
1. Fibonacci at position n 
2. Fibonacci upto nth terms 
3. Fibonacci upto n value

### CODE REVIEW

File: fibonacci-sequence/java/Fibonacci.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    public static void main(String[] args) {
+        int n = 20;
+
+        System.out.println("Fibonacci at position n = " + getFibonacciAt(n));
+        System.out.println("Fibonacci upto nth terms = " + getFibonacci(n));
+        System.out.println("Fibonacci upto n value = " + getUptoFibonacci(n));
+    }
+}
```
Comment (t2013anurag) [inline fibonacci-sequence/java/Fibonacci.java]: add a new line here.

File: fibonacci-sequence/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+-
+The sequence can be written in mathematical form as
+
+``` 
+x(n) = x(n-1) + x(n-2)
+``` 
+
+Developers Guide
+---
+
```
Comment (t2013anurag) [inline fibonacci-sequence/README.md]: remove this.

File: fibonacci-sequence/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+References:
+---
+[Fibonacci number] (https://en.wikipedia.org/wiki/Fibonacci_number)
+
+
+
```
Comment (t2013anurag) [inline fibonacci-sequence/README.md]: remove extra lines.

**Comment by t2013anurag** [comment]
@sagarduwal  - can you move the code inside the existing fibonacci directory rather than creating a new one?

**Comment by sagarduwal** [comment]
@t2013anurag  sure

**Comment by t2013anurag** [comment]
@sagarduwal - file already exists. Closing.

**Comment by sagarduwal** [comment]
implementation not available in the current file.

**Comment by t2013anurag** [comment]
can you put all your code inside fibonacci/extra_implementation or something.

**Pull request closed without merging**